routes/middlewares/audioUplodadToAWS.js

```diff
@@ -0,0 +1,32 @@
+const multer = require("multer");
+const multerS3 = require("multer-s3");
+const s3 = require("../../configs/awsS3");
+
+const storage = multerS3({
+  s3: s3,
+  bucket: process.env.AWS_BUCKET_NAME,
+  key: function (req, file, callback) {
+    try {
+      const fileType = file.mimetype.split("/")[0] !== "audio";
+
+      if (fileType) {
+        return callback(new Error("Only audio are allowed"));
+      }
+
+      const fileNameArray = file.originalname.split(".");
+      callback(
+        null,
+        "audio" + Date.now() + "." + fileNameArray[fileNameArray.length - 1]
+      );
+    } catch (err) {
```

Comment:
```suggestion
    } catch(err) {
```
catch(err)를 붙이는걸로 사용하시나요 아니면 중간 공백을 주시나요 이부분도 통일성 있게 가져가면 좋을것 같아요!!
지수님, 수인님 생각은 어떠신가요??

Reply:
저도 띄우는게 나을것 같습니다!

Reply:
저도 원래 띄워서 쓰는데 저기 잘못 작성했나봐요 수정하겠습니다!